robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -170,7 +172,8 @@ public InstrumentationConfiguration build() {
           "org.specs2",  // allows for android projects with mixed scala\java tests to be
           "scala.",      //  run with Maven Surefire (see the RoboSpecs project on github)
           "kotlin.",
-          "com.almworks.sqlite4java" // Fix #958: SQLite native library must be loaded once.
+          "com.almworks.sqlite4java", // Fix #958: SQLite native library must be loaded once.
+          "org.powermock"
```

Comment:
without this, I get

	java.lang.IllegalAccessError: tried to access field org.robolectric.RobolectricTestRunner$HelperTestRunner.this$0 from class org.robolectric.RobolectricTestRunner$HelperTestRunner$1

		at org.robolectric.RobolectricTestRunner$HelperTestRunner$1.evaluate(RobolectricTestRunner.java:545)
		at org.powermock.modules.junit4.rule.PowerMockStatement$1.run(PowerMockRule.java:65)
		at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.lang.reflect.Method.invoke(Method.java:497)
		at org.powermock.reflect.internal.WhiteboxImpl.performMethodInvocation(WhiteboxImpl.java:1873)
		at org.powermock.reflect.internal.WhiteboxImpl.doInvokeMethod(WhiteboxImpl.java:773)
		at org.powermock.reflect.internal.WhiteboxImpl.invokeMethod(WhiteboxImpl.java:638)
		at org.powermock.reflect.Whitebox.invokeMethod(Whitebox.java:401)
		at org.powermock.classloading.ClassloaderExecutor.execute(ClassloaderExecutor.java:98)
		at org.powermock.classloading.ClassloaderExecutor.execute(ClassloaderExecutor.java:78)
		at org.powermock.modules.junit4.rule.PowerMockStatement.evaluate(PowerMockRule.java:57)
		at org.robolectric.RobolectricTestRunner$2.evaluate(RobolectricTestRunner.java:255)
		at org.robolectric.RobolectricTestRunner.runChild(RobolectricTestRunner.java:185)
		at org.robolectric.RobolectricTestRunner.runChild(RobolectricTestRunner.java:54)
		at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
		at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
		at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
		at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
		at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
		at org.robolectric.RobolectricTestRunner$1.evaluate(RobolectricTestRunner.java:151)
       ...

Even though `packagesToNotAquire` is configurable externally, I guess it makes sense to include powermock by default, it wouldn't hurt, right?